data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,468 @@ mission "Coalition License Patch"
+			`	About halfway across, Tumub gets up, having caught his breath, and heads to the sail. Standing up on two legs, he opens up the sail, and skillfully manages it using the ropes, altering the form of the sail to better adapt to the changing needs of the turbulent vessel.`
+			`	As you and Pairri cheer for the group, you notice that not everyone is having the same success at crossing. Two rafts have gone dangerously too far down the river, not being able to fight the stream, and the Heliarch boats head to them at full speed, rescuing the competitors before anything drastic happens.`
+			`	In total, eighteen teams have made it across the river, with Ulmodi, Tumub and their Kimek partner winning second place by a hair, seeing as almost everyone arrived at the other margin roughly at the same time. As the competitors themselves celebrate with their own teams, and later gather all together in a large group to also applaud their opponents, the crowd is ecstatic, as praise heard in all three Coalition languages echoes all over.`
+			`	Everyone grows quiet, however, as three Heliarch Punishers come down for landing right in front of the athletes. Out come who you assume are the captains of each ship, Heliarchs wearing the same type of circlet you saw in the Ring of Friendship, with each of them being from a different Coalition species.`
+			`	"Deemed worthy, some have been. The winners, the Heliarchs shall now announce," Pairri says. As the Heliarchs start speaking she tries to summarize it all and let the box translate it, but the device quickly becomes overwhelmed. Clearly whatever it is that they are saying must be some sort of grand finale speech, judging by the way the crowd responds during the few pauses.`
+			`	After they finish, the monitor switches to a black screen, which after a moment lights up with live footage of one of the athletes, a Kimek. Then a Saryd, another Kimek, an Arach. In the end both Tumub's and Ulmodi's faces also show in the monitor, and Pairri does what you think is the Kimek version of an excited scream.`
+			`	"Selected, my friends have been!" She says. "Made it, they have!"`
+			`	There's a few more words said after everyone has seen the results and settled down, and the six that were selected are invited into the Punishers. Pairri explains that they are being considered to be elevated to the rank of Heliarch, and will go to the Ring of Friendship, to continue their journey to it.`
+			`	The ring platforms head back to the building, and as you and Pairri say your goodbyes, you take the train back to the previous arena, and from there to where your ship is docked at.`
```

Comment:
```suggestion
			`	The ring platforms head back to the building, and you and Pairri say your goodbyes. You take the train back to the previous arena and from there to where your ship is docked at.`
```
(Personal preference.) This ending is rather abrupt, especially considering how long this mission is. You might want to consider adding a sentence or two (maybe some dialogue for Pairri?) to make this feel like more of a conclusion.

Reply:
added some stuff